CMake/Dependencies/libopenssl-CMakeLists.txt

```diff
@@ -2,25 +2,29 @@ cmake_minimum_required(VERSION 2.8)
 
 project(libopenssl-download NONE)
 
-find_program(MAKE_EXE NAMES make)
-
-if (BUILD_STATIC)
-  SET(OPENSSL_EXTRA ${OPENSSL_EXTRA} no-shared no-dso)
-endif()
-
-if (DEFINED BUILD_OPENSSL_PLATFORM AND NOT BUILD_OPENSSL_PLATFORM STREQUAL OFF)
-  SET(CONFIGURE_COMMAND ${CMAKE_CURRENT_BINARY_DIR}/build/src/project_libopenssl/Configure ${OPENSSL_EXTRA} --prefix=${OPEN_SRC_INSTALL_PREFIX} --openssldir=${OPEN_SRC_INSTALL_PREFIX} ${BUILD_OPENSSL_PLATFORM})
+if (WIN32)
+  find_program(MAKE_EXE NAMES nmake)
+  SET(CONFIGURE_COMMAND perl ${CMAKE_CURRENT_BINARY_DIR}/build/src/project_libopenssl/Configure VC-WIN64A no-asm --prefix=${OPEN_SRC_INSTALL_PREFIX} --openssldir=${OPEN_SRC_INSTALL_PREFIX})
 else()
-  SET(CONFIGURE_COMMAND ${CMAKE_CURRENT_BINARY_DIR}/build/src/project_libopenssl/config ${OPENSSL_EXTRA} --prefix=${OPEN_SRC_INSTALL_PREFIX} --openssldir=${OPEN_SRC_INSTALL_PREFIX})
+  find_program(MAKE_EXE NAMES make)
+  if (BUILD_STATIC)
+    SET(OPENSSL_EXTRA ${OPENSSL_EXTRA} no-shared no-dso)
+  endif()
+
+  if (DEFINED BUILD_OPENSSL_PLATFORM AND NOT BUILD_OPENSSL_PLATFORM STREQUAL OFF)
+    SET(CONFIGURE_COMMAND ${CMAKE_CURRENT_BINARY_DIR}/build/src/project_libopenssl/Configure ${OPENSSL_EXTRA} --prefix=${OPEN_SRC_INSTALL_PREFIX} --openssldir=${OPEN_SRC_INSTALL_PREFIX} ${BUILD_OPENSSL_PLATFORM})
```

Comment:
Some of the cmake settings could be deduced from the old install-script though, not sure how useful those could be in this context